src/packageurl/contrib/django_models.py

```diff
@@ -75,7 +76,8 @@ class PackageURLMixin(models.Model):
         help_text=_('Version of the package.'),
     )
 
-    qualifiers = models.CharField(
+    # Use  models.JSONField once Django 3.1 is officially released
```

Comment:
Can you have a try/except to import `models.JSONField` and fallback to the postgres one?

Reply:
It won't hurt, but it's almost always going to fallback on the postgres one since `models.JSONField` is not even released yet. Btw I have  a ticket at https://github.com/package-url/packageurl-python/issues/27  for that.